cortex/export/export.py

```diff
@@ -0,0 +1,202 @@
+
+    list_surfaces: list of str
+        Surfaces to be used. Should be of length one, or of the same length as
+        `list_angles`. Choices are:
+            'inflated', 'flatmap', 'fiducial', 'inflated_cut',
+            or a custom dictionary of parameters.
+
+    viewer_params: dict
+        Parameters passed to the viewer.
+
+    size: tuple of int
+        Size of produced image (before trimming).
+
+    trim: bool
+        Whether to trim the white borders of the image.
+
+    sleep: float > 0
+        Time in seconds, to let the viewer open.
+
+    Returns
+    -------
+    file_names: list of str
+        Image paths.
+    """
+    msg = "list_angles and list_surfaces should have the same length."
+    assert len(list_angles) == len(list_surfaces), msg
+
+    # Create viewer
+    handle = cortex.webshow(volume, **viewer_params)
+    # Wait for the viewer to be loaded
+    time.sleep(sleep)
+
+    file_names = []
+    for view, surface in zip(list_angles, list_surfaces):
+        if isinstance(view, str):
+            if view == 'flatmap' or surface == 'flatmap':
+                # force flatmap correspondence
+                view = surface = 'flatmap'
+            view = angle_view_params[view]
+        if isinstance(surface, str):
+            surface = unfold_view_params[surface]
+
+        # Combine view parameters
+        this_view_params = default_view_params.copy()
+        this_view_params.update(view)
+        this_view_params.update(surface)
+        print(this_view_params)
+
+        # apply params
+        handle._set_view(**this_view_params)
+
+        # wait for the view to have changed
+        for _ in range(100):
+            for k, v in this_view_params.items():
+                k = k.format(subject=volume.subject) if '{subject}' in k else k
+                if handle.ui.get(k)[0] != v:
+                    print('waiting for', k, handle.ui.get(k)[0], '->', v)
+                    time.sleep(0.1)
+                    continue
+            break
+        time.sleep(0.1)
+
+        # Save image, store file_name
+        file_name = file_pattern.format(base=base_name, view=view,
+                                        surface=surface)
+        file_names.append(file_name)
+        handle.getImage(file_name, size)
+
+        # Wait for browser to dump file, before applying new view parameters
+        while not os.path.exists(file_name):
+            pass
+        time.sleep(0.1)
+
+        # Trim white edges
+        if trim:
+            try:
+                import subprocess
+                subprocess.call(["convert", "-trim", file_name, file_name])
+            except Exception as e:
+                print(str(e))
+                pass
+
+    # Try to close the window
+    try:
+        handle.close()
+        handle.server.stop()
+    except Exception as e:
+        print(str(e))
+        print('Could not close viewer.')
+
+    return file_names
+
+
+default_view_params = {
```

Comment:
I can see myself wanting to change this (and the following) view parameters. Can these be kwargs with default arguments? (otherwise one would have to modify the python files directly for finer control).

Reply:
Do you have a preference between adding a `default_view_params` parameter, and doing
```py
cortex.export.export.default_view_params['camera.azimuth'] = 46
```
?

Reply:
oh, I didn't know that would work. Then I think it's OK to leave it without extra kwargs.

Reply:
configuration state should be specified in options.cfg

for any dynamic config modification it would be preferable to just pass these things as arguments to the function when the function is called

Reply:
sorry, actually I should have read the code better. I see that you can pass a dictionary of arguments instead of those default views. So I think you can ignore my comment

Reply:
Yes, I tried to have an easy API for classic views, but something easy to customize if someone wants to go further. I am happy to improve it if you have suggestion !